src/template/helpers/debug.ts

```diff
@@ -0,0 +1,11 @@
+import { EOL } from 'os';
+
+import handlebar from 'handlebars';
+
+const regex = new RegExp(EOL, 'g');
+
+handlebar.registerHelper('$debug', function (value: unknown) {
```

Comment:
Would be nice a little comment on what the debug is doing, I guess it is adding the `value` to the artifact